README.md

```diff
@@ -204,6 +203,8 @@ The requirements for the prebuilt receipt model are slightly different.
 * For PDF and TIFF, only the first 200 pages are processed (with a free tier subscription, only the first two pages
 are processed).
 
+For tips and known issues with managing Form Recognizer models via MLOps and deployment pipelines, view [MLOps Tips and Tricks for Form Recognizer](FormRecognizer/MLOpsTips.md).
```

Comment:
Please move this above Example Scenarios and add markdown heading ""## Tips and Best Practices for Form Recognizer" - thank you @mover333